src/sentry/static/sentry/app/components/loadingError.tsx

```diff
@@ -18,9 +21,9 @@ class LoadingError extends React.Component {
   }
 
   render() {
-    const {className, message, onRetry} = this.props;
+    const {message, onRetry} = this.props;
     return (
-      <div className={classNames('alert alert-error alert-block', className)}>
```

Comment:
We probably want to keep this so it can be styled

Reply:
I didn't see any place where we passed in `className`

Reply:
OK, if we don't re-style it, then 👍 	on removing it